bin/move-node.js

```diff
@@ -5,12 +5,23 @@ const { removeNode } = require('../src/remove-node');
 
 (async () => {
   try {
-    const removedNodes = await moveNode();
+    let toNode;
+    if (process.argv.length > 2) {
```

Comment:
I think move-node should not take more than one arg. We should move one node at a time.

Reply:
Handled.